### DISCUSSION TITLE
Filters for Fields that contain multiple values lists all the values as one Filter checkbox instead of separate ones

### DISCUSSION THREAD

**Discussion by shawn-fletcher** [microsoft-search/pnp-modern-search#2488] [Q&A] [answered]
I have a document library with a Managed Metadata column for Department.   Documents can get tagged as relevant to multiple departments.   It looks like this:

![image](https://user-images.githubusercontent.com/92395989/198114553-0bc1df19-c2aa-4249-9a5d-2c29d42befc3.png)

I'd like the Search Filter to show each individual department so if it is checked, it will show any documents tagged with that department.  However, the Search Filter is actually showing all of the multiple departments as 1 checkbox for each unique instance.

For example, 2 documents have both Human Resources and Internal IT as values, so the Search Filter is showing both of the departments as 1 filter checkbox with a result count of 2.

Also, another document has 5 departments chosen, so those 5 are only 1 Filter checkbox instead of being 5 separate options.

![image](https://user-images.githubusercontent.com/92395989/198115322-6954c1df-71b3-469a-9b6e-508498c06986.png)

Any ideas on how to get these separated out?

**Comment by wobba** [comment]
Can you share how you have mapped the data and also the field/column settings for your metadata column in SharePoint.

**Comment by shawn-fletcher** [comment, marked as answer]
Thanks for the response!

Here are the column settings in the library:

![image](https://user-images.githubusercontent.com/92395989/199262727-f51c201b-334c-451f-b9b6-a40039024f45.png)

![image](https://user-images.githubusercontent.com/92395989/199262823-4d8ddba9-06ec-4e8a-8839-543244addfa3.png)

Then in the PNP Filters web part, I edited the filters and added Department:

![image](https://user-images.githubusercontent.com/92395989/199263403-d5c10e76-6c04-4649-94e3-9efe71d01c2c.png)

**Reply by wobba** [reply]
I believe it's due to how multi-value fields are mapped to the managed property - so not an easy thing to solve unfortunately. You can test if mapping ows_colname vs owstaxidcolname makes a difference.

**Reply by shawn-fletcher** [reply]
Thank you Thank you!

I mapped a new refinablestring to use OWS_Departments  and it worked!

![image](https://user-images.githubusercontent.com/92395989/199566371-19f0fed0-c173-4438-b4ae-5b9506ef4019.png)
